### PR TITLE
feat(post): add solid background color and box shadow

### PR DESCRIPTION
**Pull request by avogel3**
Updates `css` for posts to make the background a solid color and add a box shadow
Screen Shot 2022-11-11 at 10 10 08 AM

### CODE REVIEW

File: assets/css/components.css [hunk 1]
```diff
@@ -73,8 +73,9 @@ nav.pagination a:hover {
   box-sizing: border-box;
   min-width: 35rem;
   max-width: 60%;
-  background: rgba(255, 255, 255, 0.9);
-  border: 0.2rem solid rgba(255, 255, 255, 0.9);
+  background: rgba(255, 255, 255);
+  border: 0.2rem solid rgba(255, 255, 255);
+  box-shadow: 5px 5px 7px var(--navy-light);
```
Comment (vnegrisolo) [inline assets/css/components.css:78]: nice ^ 💅

File: assets/css/components.css [hunk 2]
```diff
@@ -73,8 +73,9 @@ nav.pagination a:hover {
   box-sizing: border-box;
   min-width: 35rem;
   max-width: 60%;
-  background: rgba(255, 255, 255, 0.9);
-  border: 0.2rem solid rgba(255, 255, 255, 0.9);
+  background: rgba(255, 255, 255);
```
[the remark this answers is not in the document]
Comment (vnegrisolo) [reply assets/css/components.css:76]: image

[the remark this answers is not in the document]
Comment (avogel3) [reply assets/css/components.css]: @vnegrisolo I updated it with no transparency. Here's a screen shot

**Comment by vnegrisolo** [comment]
LGTM 👍

**Pull request merged**